src/executables/server.c

```diff
@@ -46,6 +46,7 @@
 #include "connection_error.h"
 #include "network.h"
 #include "environment_variable.h"
+#include "internal_tasks_worker_pool.hpp"
```

Comment:
`cubthread::initialize` function is in `thread_manager.hpp`
```suggestion
#include "thread_manager.hpp"
```